addons/ccm-aws/ccm-aws.yaml

```diff
@@ -152,9 +152,6 @@ spec:
             - --configure-cloud-routes=false
             - --cloud-config=/etc/kubernetes/cloud/cloud-config
             - --v=2
-            {{- with .Params.CCM_CONCURRENT_SERVICE_SYNCS }}
-            - --concurrent-service-syncs={{ . }}
-            {{- end }}
```

Comment:
We need this. Unfortunately we didn't find a good way to keep it across updates, so putting it back manually is needed 😅